dashboard/app/models/email_preference.rb

```diff
@@ -15,7 +15,11 @@
 #
 #  index_email_preferences_on_email  (email) UNIQUE
 #
-#
+# Notes:
+#   source: Indicates the source of the opt-in choice.
+#   kind: "0", "1", etc.  Indicates which version of the opt-in string the
+#     user agreed to, for a given source.  This value should be bumped each
+#     time the corresponding user-facing string is updated.
```

Comment:
Thanks for adding these notes!

s/kind/form_kind, so it matches the attribute.